app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -69,16 +74,25 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * A list of recent tabs to jump back to.
  *
  * @param recentTabs List of [RecentTab] to display.
+ * @param recentSyncedTab The most recent synced tab from another device. Placeholders displayed when null.
  * @param menuItems List of [RecentTabMenuItem] shown long clicking a [RecentTab].
+ * @param shouldDisplaySyncedTab Whether the synced tab card should be displayed.
  * @param onRecentTabClick Invoked when the user clicks on a recent tab.
  * @param onRecentSearchGroupClicked Invoked when the user clicks on a recent search group.
+ * @param onRecentSyncedTabClick Invoked when the user clicks on the recent synced tab.
+ * @param onSyncedTabSeeAllClick Invoked when user clicks on the "See all" button in the synced tab card.
  */
 @Composable
+@Suppress("LongParameterList")
 fun RecentTabs(
     recentTabs: List<RecentTab>,
+    recentSyncedTab: RecentTab.SyncedTab?,
     menuItems: List<RecentTabMenuItem>,
+    shouldDisplaySyncedTab: () -> Boolean,
```

Comment:
I worry that `show` as a verb here would imply an action, especially given the type of the lambda. Looking at other properties throughout the codebase with the same type shows a preference towards the interrogative case. How does `isSyncedTabDisplayed` sound, to mirror other examples?

Reply:
`isSyncedTabDisplayed` is fine. We already use `show` a bunch in `RecentVisits`. I would investigate https://github.com/mozilla-mobile/fenix/pull/24221#discussion_r825102591 first since that would just eliminate these extra parameters entirely.